app/(admin)/admin/notifications/actions/sendEmailNotification.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+'use server';
+import * as sdk from 'node-appwrite';
```

Comment:
Do you need to import _everything_?

Reply:
Unfortunately, since 'client' and 'messaging' is being import 'as sdk' this is the only way to import it correctly.